src/core/SkRecordReplay.cpp

```diff
@@ -59,6 +61,13 @@ static inline bool EnsureInitialized() {
   return !!gRecordReplayAssert;
 }
 
+bool SkRecordReplayIsReplaying() {
```

Comment:
I am wondering, why we use a different naming scheme here? Would it not make sense to have the exact same `recordreplay` namespace with `IsReplaying()`, `AreEventsDisallowed()` etc. in it?

Reply:
This was done for consistency with skia naming conventions.